File: promptodroid/app/src/main/java/xyz/cybersapien/promptodroid/ui/adapter/PromptHolder.java
package xyz.cybersapien.promptodroid.ui.adapter;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import butterknife.BindView;
import butterknife.ButterKnife;
import xyz.cybersapien.promptodroid.R;

/**
 * Created by ogcybersapien on 1/1/18.
 */

public class PromptHolder extends RecyclerView.ViewHolder {

    @BindView(R.id.prompts_title_text_view)
    public TextView promptTitleTextView;
    @BindView(R.id.words_count_text_view)
    public TextView wordsCountTextView;
    @BindView(R.id.prompts_date_text_view)
    public TextView promptsDateTextView;

    public PromptHolder(View itemView) {
        super(itemView);
        ButterKnife.bind(this, itemView);
    }
}

File: promptodroid/app/src/main/java/xyz/cybersapien/promptodroid/ui/adapter/PromptHolder.kt
package xyz.cybersapien.promptodroid.ui.adapter

import android.support.v7.widget.RecyclerView
import android.view.View
import android.widget.TextView
import kotlinx.android.synthetic.main.item_prompt_story.view.*

/**
 * Created by ogcybersapien on 15/1/18.
 */
class PromptHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    val promptTitleTextView: TextView = itemView.prompts_title_text_view
    val wordsCountTextView: TextView = itemView.words_count_text_view
    val promptsDateTextView: TextView = itemView.prompts_date_text_view
}